src/users/repositories/users-repository-typeorm.adapter.ts

```diff
@@ -25,10 +26,9 @@ export class UsersRepositoryTypeOrmAdapter extends UsersRepository {
     return this.usersRepository.save(user);
   }
 
-  async update(username: string, user: User): Promise<User> {
-    throw Error('NotImplemented');
+  async update(username: string, payload: UpdateUserDto): Promise<void> {
+    await this.usersRepository.update(username, { username, ...payload });
```

Comment:
```suggestion
    await this.usersRepository.update(username, payload );
```
Je pense que comme le username ne peut pas changer, c'est plutot comme ca. Sinon c'est pas mal !

Reply:
Dans la logique oui mais je sais pas si ça marche ou si y faut pas passer un User complet

Reply:
Oui je lui ai dit exactement pareil car elle m'avait surpris en changeant ca (j'l'ai terrorisée en débarquant sur Teams direct :') ) mais t'as l'air d'avoir fait pareil sur books donc on a conclu que c'etait ok 🤔. A tester en somme ?

Reply:
Bon je viens de check pour book ça marche

Reply:
ok alors pour le coup j'avais repris comme dans update pour le type orm de books, mais je suis d'accord avec la suggestion d'Amine 
![image](https://user-images.githubusercontent.com/46024808/114283862-5fc6fb00-9a4c-11eb-9cd5-9146445cc14c.png)

Reply:
Yep !

Reply:
je le fais ou pas pr book?

Reply:
Si tu te sens l'envie vas-y, t'as bien uniformisé plein de choses sur le proj' déjà merci !